app/models/course.rb

```diff
@@ -5,7 +5,17 @@ class Course < ActiveRecord::Base
   has_many :quizzes, -> { order(:created_at) }
   has_many :enrollments
 
-  validates :description, presence: true
+  validates :title, :description, :price_cents, :min_quiz_count, presence: true
+  validates :min_quiz_count,
+            numericality: {
+              only_integer: true,
+              greater_than_or_equal_to: 3
+            }
+  validates :price_cents, numericality: {
+                            only_integer: true,
+                            greater_than: 0
+                          }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.